usaspending_api/etl/es_etl_helpers.py

```diff
@@ -750,7 +747,7 @@ def get_deleted_award_ids(client, id_list, config, index=None):
         for v in values_generator:
             body = filter_query(column, v)
             response = client.search(index=index, body=json.dumps(body), size=config["max_query_size"])
-            if response["hits"]["total"] != 0:
+            if response["hits"]["total"]["value"] != 0:
```

Comment:
Tricky, and annoying. 

Looks like you captured a lot of these (36 maybe), but running this from the command line has a different count found on master branch. 

Maybe try to check your branch again with this to be sure all are caught.
```
git grep -Hni "\[['\"]hits['\"]\]\[['\"]total['\"]\]"
```

Or more loosely:
```
git grep -Hni "\[['\"]hits['\"]\]"   
```

Looks like there may be a few others left in your branch that ended in newlines

```
usaspending_api/etl/es_etl_helpers.py:703:    start_ = client.search(index=index)["hits"]["total"]
usaspending_api/etl/es_etl_helpers.py:725:    end_ = client.search(index=index)["hits"]["total"]
```